scopes/ui-foundation/no-components/no-components.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { ExternalLink } from '@teambit/ui.external-link';
+import styles from './no-components.module.scss';
+
+export type NoComponentsProps = {
+  name: string;
+} & React.HTMLAttributes<HTMLDivElement>;
+
+/**
+ * page to be shown when workspace/scope has no components
+ */
+export function NoComponents({ name, children }: NoComponentsProps) {
+  return (
+    <div className={styles.noComponents}>
+      <div className={styles.title}>
+        <span>No components in</span>&nbsp;
+        <span>{name}</span>
+      </div>
+      <img src="https://static.bit.dev/harmony/no-components.svg" />
+      {children}
+      <div className={styles.bottomText}>
+        Find more info in the&nbsp;
```

Comment:
typo - Find more info **at** the..

Reply:
it looks broken because of the link after it. it's "find more in the docs" but the docs is a link so its ok